config/Config-images.in

```diff
@@ -224,6 +224,14 @@ menu "Target Images"
 		help
 		  If you don't know, 5 seconds is a reasonable default.
 
+	config EFI_IMAGES
+		bool "Build EFI GRUB images (Linux x86 or x86_64 host only)"
+		depends on TARGET_x86_64 || (TARGET_x86 && !TARGET_x86_rdc)
+		depends on TARGET_ROOTFS_EXT4FS || TARGET_ROOTFS_ISO || TARGET_ROOTFS_JFFS2 || TARGET_ROOTFS_SQUASHFS
+		select GRUB_IMAGES
```

Comment:
On ``target/linux/x86/image/Makefile``, there is a statement that ``TARGET_IMAGES_PAD`` is forced.

``TARGET_IMAGES_PAD`` can be selected here instead of forcing ``TARGET_IMAGES_PAD=y``